tests/e2e/improved_csi_idempotency.go

```diff
@@ -396,7 +396,7 @@ func createVolumesByReducingProvisionerTime(namespace string, client clientset.I
 
 	ginkgo.By("Waiting for all claims to be in bound state")
 	persistentvolumes, err = fpv.WaitForPVClaimBoundPhase(client, pvclaims,
-		framework.ClaimProvisionTimeout)
+		(2 * framework.ClaimProvisionTimeout))
```

Comment:
you don't need the parenthesis that you are adding here

Reply:
Fixed